phoenix-core/src/test/java/org/apache/phoenix/util/PhoenixRuntimeTest.java

```diff
@@ -165,6 +165,9 @@ public void testGetPkColsDataTypes() throws Exception {
                     // JIRA - https://issues.apache.org/jira/browse/PHOENIX-1329
                     continue;
                 }
+                if (!pType.canBePrimaryKey()) {
```

Comment:
Why is this check added ? Can you add a comment here.

Reply:
JSON type can't be used as primary Key.